src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/prototype/SimpleStrandSwitchVariantDetector.java

```diff
@@ -77,6 +83,8 @@ private static AlignedContig removeOverlap(final AlignedContig contig) {
         } else {
             final AlignmentInterval one = contig.alignmentIntervals.get(0),
                                     two = contig.alignmentIntervals.get(1);
+            // js is for "the shoot-off reference location of a jump that linked to alignment intervals", and
```

Comment:
Is this supposed to be "linked two alignment intervals"?

I'm not sure I understand what "shoot-off" and "landing" are -- or, I guess I can imagine, but maybe something like "jumpStart" and "jumpEnd" might be more intuitive?

Reply:
!OoO!, yes, it should be "two" instead of "to".

I have in mind a cartoonish depiction of a tiny anime figure sliding along the reference guided by the alignments of the read, sometimes jumping back and forth. Sorry for this too-pictorial naming scheme.